sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -623,7 +627,11 @@ private void onVisitChanged(VisitDto existingVisit, Visit newVisit) {
 		}
 
 		if (newVisit.getCaze() != null) {
-			caseService.updateFollowUpUntilAndStatus(newVisit.getCaze());
+			// Update case symptoms
+			CaseDataDto caze = CaseFacadeEjb.toDto(newVisit.getCaze());
+			SymptomsDto caseSymptoms = caze.getSymptoms();
+			SymptomsHelper.updateSymptoms(toDto(newVisit).getSymptoms(), caseSymptoms);
+			caseFacade.saveCase(caze);
```

Comment:
Please also call SymptomsHelper.updateIsSymptomatic before saving the case

Reply:
OK

Reply:
https://github.com/hzi-braunschweig/SORMAS-Project/blob/482e8a9d7413a5ca591ffafdde4f29544077816c/sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java#L1267-L1272

I realise `SymptomsHelper.updateIsSymptomatic` is called in `saveCase` too.

Reply:
Alright, makes sense.